tomcat-9.0/src/main/java/org/dogtagpki/tomcat/JSSContext.java

```diff
@@ -63,9 +65,14 @@ public void init(KeyManager[] kms, TrustManager[] tms, SecureRandom sr) throws K
         logger.debug("JSSContext.createSSLEngine()");
         javax.net.ssl.SSLEngine eng = ctx.createSSLEngine();
 
+	TomcatJSS instance = TomcatJSS.getInstance();
+
         if (eng instanceof JSSEngine) {
             JSSEngine j_eng = (JSSEngine) eng;
             j_eng.setCertFromAlias(alias);
+	    if(instance != null) {
+	        j_eng.setListeners(instance.getSocketListeners());
+	    }
```

Comment:
Mixed tabs/spaces here?

Reply:
Yeah something funky is happening, will fix.